### PR TITLE
Add Ampere's Case Study for LinuxBoot on Mt. Jade platform

### PR DESCRIPTION
**Pull request by nhivp**
The Ampere Mt. Jade platform exemplifies how LinuxBoot can transform traditional EDK2 firmware, making it open, flexible, and efficient. With all code publicly available, it stands as a powerful case for open source firmware adoption in modern server architectures.

### CODE REVIEW

File: src/case_studies/Ampere_study.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+Ampere has implemented and successfully upstreamed a solution for integrating LinuxBoot
+as a Boot Device Selection (BDS) option into the TianoCore EDK2 framework, as seen in
+commit
+[ArmPkg: Implement PlatformBootManagerLib for LinuxBoot](https://github.com/tianocore/edk2/commit/62540372230ecb5318a9c8a40580a14beeb9ded0).
+This innovation simplifies the boot process for the Mt. Jade platform and aligns with
+LinuxBoot's goals of efficiency and flexibility.
+
+Unlike the earlier practice that replaced the UEFI Shell with a LinuxBoot flashkernel,
+Ampere's solution introduces a custom BDS implementation that directly boots into the
```
Comment (rminnich) [inline src/case_studies/Ampere_study.md:46]: you are the first! This was always the dream!

**Review by rminnich — COMMENTED** [review]
This is really great stuff. It is also a major step forward, with your BDS implementation. 

I love the figure. I wonder if you can comment at some point about how many DXEs there are in non-linuxboot implementation? We typically removed hundreds ...

Thanks for this fantastic writeup.

**Review by rminnich — APPROVED** [review]

**Review by orangecms — APPROVED** [review]
Awesome, thank you! :book:

**Comment by nhivp** [comment]
> This is really great stuff. It is also a major step forward, with your BDS implementation.
> 
> I love the figure. I wonder if you can comment at some point about how many DXEs there are in non-linuxboot implementation? We typically removed hundreds ...
> 
> Thanks for this fantastic writeup.

Hi @rminnich, the non-linuxboot (EDK2) implementation includes ~309 PEI/DXE modules.

**Pull request merged**